package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "xo && ava"
```

Comment:
AVA runs tests in parallel and it allows us to write tests in ES2015 with `async`/`await`.